initrd-trustee_plus/scripts/aa-client-service.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+. /lib/dracut-lib.sh
+set -x
+
+# Configuration
+KBS_URL="http://10.72.140.122:31968"
```

Comment:
This highly depends where the kbs is located, is there a way to keep it more generic?

Reply:
It will be actually great if we could fetch this directly from ignition